compliance/repository/src/test/java/org/eclipse/rdf4j/repository/sparql/federation/SparqlFederatedServiceIntegrationTest.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Eclipse RDF4J contributors.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *******************************************************************************/
+package org.eclipse.rdf4j.repository.sparql.federation;
+
+import org.eclipse.rdf4j.repository.Repository;
+import org.eclipse.rdf4j.repository.sail.SailRepository;
+import org.eclipse.rdf4j.sail.memory.MemoryStore;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+public class SparqlFederatedServiceIntegrationTest {
+
+	@Test
+	@Disabled("manual test to demonstrate the original issue of GH-5358")
```

Comment:
if prefered, I can also delete this test class. It is just one example to quickly run check whether the fix is working. This one was previously throwing an exception